stac_fastapi/extensions/stac_fastapi/extensions/core/context.py

```diff
@@ -1,4 +1,5 @@
 """Context extension."""
```

Comment:
The context extension is deprecated! I think we can 🔪 it!

Reply:
maybe in another PR

Reply:
https://github.com/stac-utils/stac-fastapi/issues/649